README.md

```diff
@@ -1 +1,103 @@
-# uploadcare-netlifycms
\ No newline at end of file
+# NetlifyCMS Uploadcare plugin
+
+<a href="https://uploadcare.com/?utm_source=github&utm_campaign=uploadcare-netlifycms">
+    <img align="right" width="64" height="64"
+      src="https://ucarecdn.com/2f4864b7-ed0e-4411-965b-8148623aa680/uploadcare-logo-mark.svg"
+      alt="">
+    <img align="right" width="64" height="64"
+      src="https://www.netlify.com/img/press/logos/logomark.svg"
+      alt="">
+</a>
+
+This is Uploadcare plugin for [Netlify CMS]. It will allow
+your users to upload files and images from local device, social networks, cloud
+storages without any backend code that is usually required to handle uploads.
+
+[![NPM version][npm-img]][npm-url]
+[![GitHub release][badge-release-img]][badge-release-url]&nbsp;
+[![Uploadcare stack on StackShare][badge-stack-img]][badge-stack-url]
+
+# Demo
+
+A demo based on [netlify-templates/one-click-hugo-cms][one-click-hugo-cms] can be found here: [Demo]
+
+Registration is open but [Git Gateway] disabled so you can't publish your changes.
+
+## Installation & configuration
```

Comment:
We need three pats

```
## Install

....

## Usage

...

## Configuration

....

```

Like in https://github.com/uploadcare/uploadcare-ckeditor/ or https://github.com/uploadcare/uploadcare-redactor/